documentation/meta/traffic/runbooks/identifying-and-blocking-traffic-anomalies.md

```diff
@@ -0,0 +1,161 @@
+2.  Select the relevant domain in the Cloudflare UI.
+3.  Go to `Analytics & Logs => Traffic` and choose the "Page Views" box in the
+    left sidebar.
+4.  Set the time range dropdown to 24 hours.
+5.  See if you can find the point where the current spike in traffic begins.
+6.  If you do not see a place where traffic stabilizes to a relatively flat
+    line, continue increasing the time range in the dropdown.
+7.  Once you find where normalized traffic starts to spike in the graph, hover
+    over the graph and **record the date and time the spike began**.
+
+### 2. Identify targetable sources of the suspicious traffic
+
+Next, we need to associate the traffic with specific sources. Essentially, we're
+trying to find unique identifiers linked to potentially malicious traffic, so we
+can block them through the Cloudflare Web Access firewall.
+
+There are three primary identifiers for targeting traffic. These are ordered
+from largest, most persistent, and most impactful, to smallest and most
+ephemeral. When possible, **blocking ASNs** is the single best way to mitigate
+malicious traffic.
+
+#### Traffic Source Identifiers
+
+- Source ASNs (Autonomous System Numbers): Unique identifiers for a group of IP
+  networks and routers under the control of a single organization, typically an
+  internet service provider or hosting company.
+
+  Typically, a single ASN makes up less than 10% of total Openverse requests in
+  a given timeframe. If this percentage is larger, it could suggest that a user
+  on the ASN is scraping Openverse or accessing Openverse programmatically in an
+  aggressive way.
+
+- Source user agents: Unique user-agent strings sent with requests to Openverse.
+  Nearly all UA strings from web browsers begin with `Mozilla`. Outside of the
+  latest stable version of Chrome on Windows 10, which is the most popular
+  Openverse browser (and usually looks something like
+  `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/{webkit-version} (KHTML, like Gecko) Chrome/{real-chrome-version} Safari/{webkit-version}`),
+  no single user agent should take up more than 10% of total traffic. Excluding
+  these with a `Source user agent does not contain Mozilla` or
+  `Source browser equals Unknown` filter in the analytics UI can reveal other
+  user agents from things like social media platforms and popular http request
+  software libraries, like `Go-http-client/2.0` or `axios/0.21.4`, as examples.
+
+- IP addresses: A unique identifier for a single machine or user.
+
+  Typically, a single IP address makes up less than 1% of total Openverse
+  requests in a given timeframe. These larger IPs typically belong to "good" web
+  crawlers from places like Google, Bing, and Dotbot, as examples. If this
+  percentage is larger, it could suggest that a user on the ASN is scraping
+  Openverse or accessing Openverse programmatically in an aggressive way.
+
+#### Steps
+
+1.  Using the view from the previous task, scroll down and look for IP
```

Comment:
Helpful cross-reference. The "previous task" is ambiguous and the section immediately preceding this one is non-trivial. It would be easy to get lost (I did!)

```suggestion
1.  Using [the view from the previous task](#1-identify-the-timeframe-of-the-traffic-anomalies), scroll down and look for IP
```